Solutions/CohesitySecurity/Data Connectors/Helios2Sentinel/zip.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/zsh
```

Comment:
Consider renaming the files from
zip.sh -> publish_zip.sh
zip.test.sh-> publish_zip.test.sh